code/src/pages/CreateUsernamePage.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import styled from 'styled-components';
+
+import { gamestate } from 'reducers/gamestate';
+import { StartGamePage } from './StartGamePage';
+
+import { StartGameButton } from 'styled-components/StartGameButton';
+
+const CreateUsernameBackground = styled.div`
```

Comment:
How could we name this variable so we indicate the relation to `UsernameFormContainer `
- It should contain `UsernameFormContainer ` in order for the background to work properly.
- It's also responsible for other styling than just the background